python/libraries/bimvee/plotFlow.py

```diff
@@ -0,0 +1,115 @@
+        fig_path {string} -- save path for the generated figure
+        fig_name {string} -- name of the generated figure
+
+    Keyword Arguments:
+        fig_subtitle {string} -- optional figure sub-title (default: {None})
+    """
+    fig = plt.figure(figsize=(16.0, 10.0))
+    if isinstance(fig_subtitle, str):
+        fig.suptitle("FLOW Events\n" + fig_subtitle, fontsize=20, fontweight='bold')
+    else:
+        fig.suptitle("FLOW Events", fontsize=20, fontweight='bold')
+
+
+    ax11 = plt.subplot(2, 1, 1)
+    plt.plot(flowDict['ts'], flowDict['vx'], color=retro_palette[0], marker='.', linewidth=0.0)
+    plt.xlabel('Time [s]', fontsize=10, fontweight='bold')
+    plt.ylabel('Vx [px/s]', fontsize=10, fontweight='bold')
+    plt.grid(True)
+
+    plt.subplot(2, 1, 2, sharex=ax11)
+    plt.plot(flowDict['ts'], flowDict['vy'], color=retro_palette[1], marker='.', linewidth=0.0)
+    plt.xlabel('Time [s]', fontsize=10, fontweight='bold')
+    plt.ylabel('Vy [px/s]', fontsize=10, fontweight='bold')
+    plt.grid(True)
+
+    # fig.tight_layout(pad=8, h_pad=2.0, w_pad=None)
+    fig.tight_layout()
+    if isinstance(fig_subtitle, str):
+        fig.subplots_adjust(top=0.9)
+    else:
+        fig.subplots_adjust(top=0.95)
+
+    fig.align_ylabels()
+
+    plt.savefig(os.path.join(fig_path, fig_name + ".png"), dpi=300, bbox_inches='tight')
+    print("Saving " + fig_name + ".png")
+    plt.close()
+
+#-----------------------------------------------------------------------------------------------------
+def plotFlowDistribution(flowDict, fig_path, fig_name, fig_subtitle=None):
+    """
+    Plot the distribution of the FLOW events in flowDict and save the generated
+    figure as fig_name.png at the location defined by fig_path.
+
+    Arguments:
+        flowDict {dict} -- dictionary of FLOW events as formatted by bimvee from event-driven library
+        fig_path {string} -- save path for the generated figure
+        fig_name {string} -- name of the generated figure
+
+    Keyword Arguments:
+        fig_subtitle {string} -- optional figure sub-title (default: {None})
+    """
+    fig = plt.figure(figsize=(14.0, 10.0))
+    if isinstance(fig_subtitle, str):
+        fig.suptitle("FLOW Events Distribution\n" + fig_subtitle, fontsize=20, fontweight='bold')
+    else:
+        fig.suptitle("FLOW Events Distribution", fontsize=20, fontweight='bold')
+    
+    plt.subplot(2,1,1)
+    sns.distplot(flowDict['vx'], bins=100, color = retro_palette[0])
+    plt.xlabel('Vx [px/s]', fontsize=10, fontweight='bold')
+
+    plt.subplot(2,1,2)
+    sns.distplot(flowDict['vy'], bins=100, color = retro_palette[1])
+    plt.xlabel('Vy [px/s]', fontsize=10, fontweight='bold')
+
+    fig.tight_layout()
+    if isinstance(fig_subtitle, str):
+        fig.subplots_adjust(top=0.9)
+    else:
+        fig.subplots_adjust(top=0.95)
+
+    plt.savefig(os.path.join(fig_path, fig_name + '.png'), dpi=300, bbox_inches='tight')
```

Comment:
These lines from here to the end could be optional - it would let us just see the plot without saving it.

Reply:
Commit e0be329 should solve this point.